Tests/Marketplace/approved_usecases.json

```diff
@@ -15,7 +15,7 @@
 		"HIPAA Breach Notification",
 		"Health Check",
 		"Hunting",
-		"Identity And Access Management",
+		"Identity Lifecycle Management",
```

Comment:
don't delete it, there are other packs that use "Identity And Access Management"

Reply:
ok, so I'll just change it to small "a" from some reason it's capital here but small anywhere else